packages/react-percy/package.json

```diff
@@ -35,7 +35,6 @@
   "dependencies": {
     "@percy-io/react-percy-ci": "^0.2.7",
     "@percy-io/react-percy-config": "^0.2.6",
-    "@percy-io/react-percy-webpack": "^0.2.7",
```

Comment:
Now @percy-io/react-percy-ci is the only package with a dependency on @percy-io/react-percy-webpack.

Reply:
Correct!